interview/infra/ocr/tesseract_client.py

```
@@ -0,0 +1,34 @@
+
+
+class TesseractOCRClient:
+    def __init__(self, lang: str = "kor+eng"):
+        self.lang = lang
+
+    def extract_text_from_image(self, image_path: str) -> str:
+        image = Image.open(image_path)
+        text = pytesseract.image_to_string(image, lang=self.lang)
+        return text.strip()
+
+    def extract_text_from_pdf(self, pdf_path: str) -> str:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)